docs/mobile-apps/live-testing/live-mobile-app-testing.md

```diff
@@ -24,30 +24,30 @@ You can upload your app via the Sauce Labs UI or via the REST API. For informati
 
 To upload an app via the Sauce Labs UI:
 
-1. On Sauce Labs, in the left panel, click **LIVE** and then click **Mobile App**.
-2. Click **Upload App**. You can either drag and drop an app, or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
+1. On Sauce Labs, in the left panel, click **App Management**.
+2. To upload an app you can either drag and drop an app or browse for and select the file. We currently support \*.apk Android app files, \*.aab Android App Bundle files and \*.ipa or \*.zip iOS app files (\*.zip files are parsed to determine whether a valid \*.app bundle exists). Non-app file uploads are not supported in the UI at this time, but can be uploaded through the API.
 
 If you don't have an app to test, you can use the [Sauce Labs sample mobile app](https://github.com/saucelabs/sample-app-mobile).
 
-<img src={useBaseUrl('img/live-testing/live-mobile-app-nav.png')} alt="Upload an app" width="650"/>
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management.png')} alt="Upload an app" width="650"/>
 
 ### Deleting an App
 
-Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
+Deleting an app in Sauce Labs will delete the whole app (i.e., the group of builds belonging to the same app package). Files associated with app identifiers (i.e., belonging to the same platform and accessible to the same team) are indicated by the + symbol next to the version number. Also, the version number shown is the most recently updated file, not necessarily the latest version of the app.
 
-To delete an app, on the Mobile App test page, hover over the test and then click **Delete**.
+To delete an app, on the App Management page, hover over the build and then click **Delete**.
 
-<img src={useBaseUrl('img/live-testing/live-mobile-app-delete.png')} alt="Delete an app" width="650"/>
+<img src={useBaseUrl('img/live-testing/live-mobile-app-management-delete.png')} alt="Delete an app" width="850"/>
 
 ### App Settings
 
-To view or change the app settings, on the Mobile App test page, hover over the app and then click **Settings**.
+To view or change the app settings, on the App Management test page, hover over the app and then click **Settings**.
```

Comment:
```suggestion
To view or change the app settings, on the **App Management** page, hover over the app and then click **Settings**.
```

Reply:
The step says to click "Settings" but the tooltip says "Settings and App Versions"